kerykeion/planetary_returns/factory.py

```
@@ -248,6 +248,62 @@ def _parse_iso(iso_formatted_time: str) -> datetime:
+        side.
+
+        Ordering between a seed and a return is therefore defined at the
+        library's reporting resolution. A forward search starts from the whole
+        second AFTER the seed's: every crossing inside the seed's own second
+        is reported at that second, hence not "after" it. A backward search
+        starts from the seed's own whole second: the ephemeris backend's
+        backward searches are strictly past (a crossing at the start epoch is
+        never the answer), so a crossing inside the seed's second — reported
+        at that second, hence not "before" it — is excluded, while one in the
+        second before is found. Reported instants become re-usable as seeds —
+        ``next`` from the instant of return N is N+1, ``previous`` is N−1,
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)